### PR TITLE
removed erroring logger for servers

### LINKED ISSUE #316: Error when deleting a colony
```
07:53] [Server thread/INFO] [minecolonies/]: Colony 183 - removed AbstractBuilding BlockPos{x=-609, y=67, z=-8469} of type TownHall
[20:07:53] [Server thread/ERROR] [Sponge/]: Error occurred while executing command 'mc colony delete 183' for source EntityPlayerMP['Kostronor'/1905714, l='minecolonies', x=-608.40, y=67.00, z=-8470.84]: net.minecraft.world.World.func_72827_u()Ljava/lang/String;
java.lang.NoSuchMethodError: net.minecraft.world.World.func_72827_u()Ljava/lang/String;
	at com.minecolonies.colony.ColonyManager.deleteColony(ColonyManager.java:126) ~[ColonyManager.class:?]
	at com.minecolonies.commands.DeleteColonyCommand.lambda$execute$61(DeleteColonyCommand.java:85) ~[DeleteColonyCommand.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at net.minecraft.server.MinecraftServer.func_175586_a(MinecraftServer.java:1347) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_152344_a(MinecraftServer.java:1359) ~[MinecraftServer.class:?]
	at com.minecolonies.commands.DeleteColonyCommand.execute(DeleteColonyCommand.java:85) ~[DeleteColonyCommand.class:?]
	at com.minecolonies.commands.AbstractSplitCommand.execute(AbstractSplitCommand.java:74) ~[AbstractSplitCommand.class:?]
	at com.minecolonies.commands.AbstractSplitCommand.execute(AbstractSplitCommand.java:74) ~[AbstractSplitCommand.class:?]
	at com.minecolonies.commands.CommandEntryPoint.func_184881_a(CommandEntryPoint.java:54) ~[CommandEntryPoint.class:?]
	at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:104) ~[k.class:?]
	at org.spongepowered.common.command.MinecraftCommandWrapper.tryExecute(MinecraftCommandWrapper.java:143) ~[MinecraftCommandWrapper.class:1.10.2-2123-5.1.0-BETA-1879]
	at org.spongepowered.common.command.MinecraftCommandWrapper.process(MinecraftCommandWrapper.java:129) ~[MinecraftCommandWrapper.class:1.10.2-2123-5.1.0-BETA-1879]
	at org.spongepowered.api.command.dispatcher.SimpleDispatcher.process(SimpleDispatcher.java:333) ~[SimpleDispatcher.class:1.10.2-2123-5.1.0-BETA-1879]
	at org.spongepowered.common.command.SpongeCommandManager.process(SpongeCommandManager.java:291) [SpongeCommandManager.class:1.10.2-2123-5.1.0-BETA-1879]
	at net.minecraft.command.ServerCommandManager.func_71556_a(SourceFile:82) [bd.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:913) [me.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:892) [me.class:?]
	at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:37) [im.class:?]
	at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:9) [im.class:?]
	at org.spongepowered.common.network.PacketUtil.onProcessPacket(PacketUtil.java:125) [PacketUtil.class:1.10.2-2123-5.1.0-BETA-1879]
	at net.minecraft.network.PacketThreadUtil$1.redirect$onProcessPacket$zjc000(SourceFile:39) [fl$1.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fl$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_101]
	at net.minecraft.util.Util.func_181617_a(SourceFile:45) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
```

### PR DESCRIPTION
**Pull request by nytstarlet**
Closes #316 

# Changes proposed in this pull request:

-  removed erroring logger for servers

Review please

### CODE REVIEW

**Review by marvin-bitterlich — APPROVED** [review]
:shipit:

File: src/main/java/com/minecolonies/coremod/colony/ColonyManager.java [hunk 1]
```diff
@@ -159,7 +159,6 @@ public static void deleteColony(final int id)
                 building.destroy();
                 for (final World world : colonyWorlds)
                 {
-                    Log.getLogger().info("Try out World " + world.getProviderName());
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/colony/ColonyManager.java:162]: does this cause any problems?

Comment (marvin-bitterlich) [reply src/main/java/com/minecolonies/coremod/colony/ColonyManager.java:162]: The method was client side only and errored on servers

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/328/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**